<launch>
  <arg name="local_ip"/>

  <node name="$(anon optitrack_node)" pkg="vision_pkg" type="streamnode" output="screen" respawn="true">

    <param name="local_ip" value="$(arg local_ip)"/>
    <param name="use_thread" value="false"/>
    <param name="publish_frequency" value="125"/>

    <!-- enable this tag to track specific objects only, otherwise it tracks everything. -->
    <!--param name="obj_list" value="myobj1_root myobj2_root"-->

    <!-- enable this to load a calibration file and get the correct /vision and /robot frames on the tf -->
    <!--param name="calib_file" value="mycalibfile.txt"-->
  </node>
</launch>
